### PR TITLE
RN 33215/36489 - BookingData

### PR DESCRIPTION
**Pull request by spetebroodt**

### CODE REVIEW

File: release-notes/Cube/Cube_Main_Release_10.4/Cube_10.4.0_features_changes.md [hunk 1]
```diff
@@ -9,6 +9,57 @@ uid: Cube_Main_Release_10.4.0_other_features_changes
...
+  - the resource name
+  - an icon indicating the function of the resource
+  - the node label or, if no node label is defined, the name of the function definition
+  - the icon of the service definition node (only if the resource to mapped to a service definition node)
+
+- On the right-hand side, you can select a node or node interface of the resource selected in the list on the left.
+
+  After selecting a node or node interface, this section will show the profile data of this node or node interface:
+
+  - the profile instance, and
```
Comment (HansSolos) [inline release-notes/Cube/Cube_Main_Release_10.4/Cube_10.4.0_features_changes.md:40]: Maybe also add something like "if any", or "if applicable"

File: release-notes/Cube/Cube_Main_Release_10.4/Cube_10.4.0_features_changes.md [hunk 2]
```diff
@@ -9,6 +9,57 @@ uid: Cube_Main_Release_10.4.0_other_features_changes
...
+  - the overridden parameter values.
```
Comment (HansSolos) [inline release-notes/Cube/Cube_Main_Release_10.4/Cube_10.4.0_features_changes.md:41]: Note that these overridden values can be done on several levels.
So eventually, the user will see "the actual used parameter values of the profile parameters'.
Should this info be taken into account?

File: release-notes/Cube/Cube_Main_Release_10.4/Cube_10.4.0_features_changes.md [hunk 3]
```diff
@@ -9,6 +9,57 @@ uid: Cube_Main_Release_10.4.0_other_features_changes
...
+  
+  > [!NOTE]
+  > Parameter value priority:
```
Comment (HansSolos) [inline release-notes/Cube/Cube_Main_Release_10.4/Cube_10.4.0_features_changes.md:44]: Could be "Parameter value override by priority:"?

File: release-notes/Cube/Cube_Feature_Release_10.3/Cube_10.3.8.md [hunk 4]
```diff
@@ -16,6 +16,57 @@ uid: Cube_Feature_Release_10.3.8
 
 ## Other new features
 
+#### Visual Overview: New BookingData component [ID_33215] [ID_36489]
+
+<!-- MR 10.4.0 - FR 10.3.8 -->
```
Comment (HansSolos) [inline release-notes/Cube/Cube_Feature_Release_10.3/Cube_10.3.8.md:21]: See other page for the changes. It looks like it is the same text/description.

File: release-notes/Cube/Cube_Main_Release_10.4/Cube_10.4.0_features_changes.md (hunk 1 above)
Comment (spetebroodt) [reply release-notes/Cube/Cube_Main_Release_10.4/Cube_10.4.0_features_changes.md:40]: corrected

**Comment by spetebroodt** [comment]
@edalslc Can you please have a look at this PR and approve if OK?

File: release-notes/Cube/Cube_Main_Release_10.4/Cube_10.4.0_features_changes.md [hunk 5]
```diff
@@ -9,6 +9,57 @@ uid: Cube_Main_Release_10.4.0_other_features_changes
...
+  After selecting a node or node interface, this section will show the profile data of this node or node interface:
+
+  - the profile instance (if applicable), and
+  - the profile parameter values that will be used (note that these values can be overridden on several levels).
+  
+  > [!NOTE]
+  > Priority of profile parameter value overrides:
+  >
+  > 1. Resource usage parameter overrides
+  > 1. Resource usage profile instance
```
Comment (edalslc) [inline release-notes/Cube/Cube_Main_Release_10.4/Cube_10.4.0_features_changes.md:47]: Values defined in profile instance

File: release-notes/Cube/Cube_Main_Release_10.4/Cube_10.4.0_features_changes.md [hunk 6]
```diff
@@ -9,6 +9,57 @@ uid: Cube_Main_Release_10.4.0_other_features_changes
...
+  > 1. Resource usage profile definition
```
Comment (edalslc) [inline release-notes/Cube/Cube_Main_Release_10.4/Cube_10.4.0_features_changes.md:48]: Values defined in profile definition

File: release-notes/Cube/Cube_Main_Release_10.4/Cube_10.4.0_features_changes.md [hunk 7]
```diff
@@ -9,6 +9,57 @@ uid: Cube_Main_Release_10.4.0_other_features_changes
...
+- On the left-hand side, you will find a list of resources used by the booking.
+
+  For every resource, this list shows the following information:
+  
+  - the resource name
+  - an icon indicating the function of the resource
+  - an icon indicating whether the resource is linked to a service definition node
+  - the node label or, if no node label is defined, the name of the function definition
+
+- On the right-hand side, you can select a node or node interface of the resource selected in the list on the left.
```
Comment (edalslc) [inline release-notes/Cube/Cube_Main_Release_10.4/Cube_10.4.0_features_changes.md:36]: for a left hand side selection, there is only one single node that you can select here. + of course the node interfaces

**Pull request merged**